modules/mqtt/mqtt-options.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2021 One Identity
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+#ifndef MQTT_CLIENT_OPTIONS_INCLUDED
+#define MQTT_CLIENT_OPTIONS_INCLUDED
+
+#include "driver.h"
+#include <MQTTClient.h>
+
+#define MQTT_DISCONNECT_TIMEOUT 10000
+
+typedef struct _MQTTClientOptions MQTTClientOptions;
+
+struct _MQTTClientOptions
+{
+  gint keepalive;
```

Comment:
No, it needs to be `gint`. Because in this case `keepalive` means how many seconds can be without "ping". 
https://www.eclipse.org/paho/files/mqttdoc/MQTTClient/html/struct_m_q_t_t_client__connect_options.html#ac8dd0930672a9c7d71fc645aa1f0521d